src/mpi/errhan/errutil.c

```diff
@@ -1656,6 +1693,10 @@ static int vsnprintf_mpi(char *str, size_t maxlen, const char *fmt_orig, va_list
                     MPL_snprintf(str, maxlen, "errh=0x%x", E);
                 }
                 break;
+            case (int) 'K':
```

Comment:
Is this needed? Or just an improvement to the error output? I can't see why returning the predefined integer values is wrong.

Reply:
It is for consistency. I don't see why the KEYVAL handle object is any different from other handle objects. The integer value wouldn't make any sense to the user -- at least we should print it as hexadecimals.